engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -2699,11 +2898,27 @@ public static int dayOfYear(@Nullable final ZonedDateTime dateTime) {
         return dateTime.getDayOfYear();
     }
 
+    /**
+     * Returns a 1-based int value of the month of the year (Julian date) for a {@link LocalDate}. January is 1,
+     * February is 2, etc.
+     *
+     * @param date date to find the month of the year of
+     * @return {@link QueryConstants#NULL_INT} if either input is {@code null}; otherwise, the month of the year
+     */
+    @ScriptApi
+    public static int monthOfYear(@Nullable final LocalDate date) {
```

Comment:
The other methods have lined up well w/ the java time API - should this be `monthValue` to also line up?

Reply:
I'm not inclined to make this change.  The current naming is intuitive and consistent with other functions like `dayOfMonth`.